vcd/datasource_vcd_edgegateway.go

```diff
@@ -97,6 +107,90 @@ func datasourceVcdEdgeGateway() *schema.Resource {
 				Computed:    true,
 				Description: "'accept' or 'deny'. Default 'deny'",
 			},
+			"fips_mode_enabled": &schema.Schema{
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "Enable FIPS mode. FIPS mode turns on the cipher suites that comply with FIPS. (False by default)",
+			},
+			"use_default_route_for_dns_relay": &schema.Schema{
+				Type:        schema.TypeBool,
+				Computed:    true,
```

Comment:
Safe to not need to explicitly specify `Default: false` as in other places?

Reply:
It is a data source so I guess it should be "read only: